pkgs/development/libraries/libarchive/default.nix

```diff
@@ -31,7 +32,7 @@ in
 assert xarSupport -> libxml2 != null;
 stdenv.mkDerivation (finalAttrs: {
   pname = "libarchive";
-  version = "3.6.2";
+  version = "3.6.2";   # please drop the patch at the bottom of the file when bumping
```

Comment:
```suggestion
```

Redundant.

Reply:
Since patch is conditional, and has to be at the bottom (or top) of the file in order to avoid rebuilds, I'm willing to bet that without this comment or an `assert`, whoever bumps the version forgets to drop the patch, and then we're right back where we started with static builds being broken...